### DISCUSSION TITLE
Files corrupt? Cannot access main.py

### DISCUSSION THREAD

**Discussion by zmaier24** [micropython/micropython#10100] [General]
Hello,

i am using a Pyboard V1.1 and since yesterday i cannot access the main.py file anymore.
All others files are fine.
Any idea how to fix this? 

Best wishes
Martin

![image](https://user-images.githubusercontent.com/63282373/204128820-750a2377-defb-4fff-94c9-81bd9d9de5b3.png)

**Comment by robert-hh** [comment]
Probably the file system is corrupt and has to be rebuilt. That happens when both the PC and the board write to the file system, or the board was not properly ejected after writing from the PC. 
To avoid trouble like that, configure the board to NOT have msc support and use mpremote to tansfer files. See pyb.usb_mode at https://docs.micropython.org/en/latest/library/pyb.html?highlight=msc

**Reply by zmaier24** [reply]
Thx for the fast reply.
So `pyb.usb_mode("VCP")` should disable the msc. Perfect, but how can i rebuild the FS? Any change to restore the file, i know backup, but my backup was propably not the last version 😉

**Reply by robert-hh** [reply]
You can recreate the file system, but all files are lost. See https://docs.micropython.org/en/latest/pyboard/tutorial/reset.html chapter 7.2.

**Reply by zmaier24** [reply]
Thank you!!!
